index.html

```diff
@@ -122,177 +68,185 @@
-
-    <a class="logo" href="#">
-      <img src="images/h-logo-white.svg" alt="logo">
-       
-    </a>
-    <p class="logo-text">
-      2022 World health Assembly. All Rights Reserved.
-      This is registred under the MIT license. for more
-      info, please <a
-        href="https://github.com/Alusp/World-Health-Assembly">See
-        here</a>
-    </p>
-
-  </footer>
-
-  <script src="script.js"></script>
-
-</body>
-
+    </header>
+
+    <main>
+      <section class="head">
+        <div class="head-bg"></div>
+        <p class="hello">"World-Health-Organization"</p>
+        <h1 class="title">
+          Seventy-fifth World Health <br />
+          Assembly <br />
+          2022
+        </h1>
+        <p class="supporting-text">
+          The World Health Assembly is the decision-making body of WHO. It is
+          attended by delegations from all WHO Member States and focuses on a
+          specific health agenda prepared by the Executive Board. The main
+          functions of the World Health Assembly are to determine the policies
+          of the Organization, appoint the Director-General, supervise financial
+          policies, and review and approve the proposed programme budget.
+        </p>
+        <div class="small-text">
+          <h2 class="date">2022.08.09 (MON)~ 04 (FRI)</h2>
+        <p class="place">@annually in Geneva, Switzerland.</p>
+        </div>
+      </section>
+      <section class="program-section">
+        <div class="title-group">
+          <h2 class="title">Main Program</h2>
+          <hr />
+        </div>
+        <ul class="programs">
+          <li>
+            <div>
+              <i class="fal fa-keynote fa-2x"></i>
+              <h3>Lecture</h3>
+            </div>
+
+            <p>
+              Proceedings will be webcast live from this web page.
+              Simultaneously available in Six Languages.
+            </p>
+          </li>
+
+          <li>
+            <div>
+              <i class="fad fa-comments-alt fa-2x"></i>
+              <h3>Forum</h3>
+            </div>
+
+            <p>
+              The sessions can be watched live on 22-28 Sept 2022. Recordings
+              will be available in the listing.
+            </p>
+          </li>
+          <li>
+            <div>
+              <i class="far fa-tools fa-2x"></i>
+              <h3>Workshop</h3>
+            </div>
+
+            <p>
+              Strengthening preparedness and response to health are key theme of
+              the Assembly.
+            </p>
+          </li>
+          <li>
+            <div>
+              <i class="fal fa-keynote fa-2x"></i>
+              <h3>WHO Ignite</h3>
+            </div>
+
+            <p>
+              The Proceedings is to enable dialog and not constitute an verbatim
+              record of the proceedings.
+            </p>
+          </li>
+        </ul>
+        <a class="see-more">SEE THE WHOLE PROGRAM</a>
+      </section>
```

Comment:
- [ ] The middle section of the Home page should contain 5 individual articles and they should be organized in a row for the desktop version. Your list only include 4 articles and they are not responsive to a row style in all desktop screen sizes. Kindly fix that.
Template:
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/61919617/181601456-42d34b1b-0dee-4a36-9066-75b23a30c347.png">